locations/spiders/pet_paradise.py

```diff
@@ -43,4 +44,5 @@ def parse(self, response):
                 "website": store_data["interestpointMoreInfoLink"],
             }
 
+            apply_category({"shop": "pet_grooming"}, properties)
```

Comment:
Could you please add this category into `Categories` enum in [categories.py](https://github.com/alltheplaces/alltheplaces/blob/master/locations/categories.py) and then use it from Categories enum?

Reply:
The website looks like it's more focused on https://wiki.openstreetmap.org/wiki/Tag:amenity=animal_boarding. This is one of the reasons I prefer the category in NSI, I have no idea about this brand, but someone in OSM will eventually fix it if it's in NSI, they wont fix it here.